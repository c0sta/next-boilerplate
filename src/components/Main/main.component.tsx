const Main = () => {
  return (
    <section>
      <h1>test</h1>
    </section>
  )
}

export default Main
