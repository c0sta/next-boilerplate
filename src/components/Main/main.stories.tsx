import Main from './main.component'
import { Story, Meta } from '@storybook/react/types-6-0'

// titulo e componente que vÃ£o aparecer na dashboard
export default {
  title: 'Main',
  component: Main,
  argTypes: {
    title: { control: 'text' },
    description: { control: 'text' }
  }
} as Meta

export const Default: Story = (args) => <Main {...args} />
