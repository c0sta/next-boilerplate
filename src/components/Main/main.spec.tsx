import { render, screen } from '@testing-library/react'
import Main from './main.component'

describe('<Main />', () => {
  it('should render test', () => {
    const { container } = render(<Main />)
    expect(screen.getByRole('heading', { name: /test/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
