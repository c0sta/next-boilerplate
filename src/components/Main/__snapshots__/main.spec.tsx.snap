// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render test 1`] = `
<section>
  <h1>
    test
  </h1>
</section>
`;
